src/pg/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub mod composite_source;
```

Comment:
Should `pg` maybe already be called `database`? Or tile source?

Reply:
I'm working on adding other backends beyond pg